test/components/generators/test_hugging_face_local_generator.py

```diff
@@ -459,3 +463,15 @@ def test_hf_pipeline_runs_with_our_criteria(self):
         results = generator.run(prompt="something that triggers something")
         assert results["replies"] != []
         assert generator.stopping_criteria_list is not None
+
+    @pytest.mark.integration
```

Comment:
I also added an integration test with a small model: it was missing and it helps to make sure everything works...